pkg/operator/ceph/config/network_test.go

```diff
@@ -144,14 +144,17 @@ func TestGenerateNetworkSettings(t *testing.T) {
 }
 
 func TestGetNetworkRange(t *testing.T) {
-	ns := "rook-ceph"
-	nad := &networkv1.NetworkAttachmentDefinition{
-		ObjectMeta: metav1.ObjectMeta{
-			Name:      "public-network-attach-def",
-			Namespace: ns,
-		},
-		Spec: networkv1.NetworkAttachmentDefinitionSpec{
-			Config: `{
+	// Simple IPAM host-local
+	{
+		t.Log("simple host-local IPAM test")
```

Comment:
How about using `t.Run()` instead of `t.Log()` and these brace blocks?

```go
t.Run("simple host-local IPAM test", func(t *testing.T) {
  // do test
})
```